src/platform/silabs/CHIPDevicePlatformConfig.h

```diff
@@ -61,6 +61,28 @@
 #define CHIP_DEVICE_CONFIG_DEFAULT_DEVICE_HARDWARE_VERSION SL_HARDWARE_VERSION
 #endif
 
+/**
+ *  Allow for some test/fall-back values to be used
+ * Production builds shall set to 0 or remove this option
+ */
+#ifndef CHIP_DEVICE_CONFIG_ENABLE_TEST_SETUP_PARAMS
+#define CHIP_DEVICE_CONFIG_ENABLE_TEST_SETUP_PARAMS 1
```

Comment:
This really should not be true by default for the platform, ideally... but I guess other platforms are doing it.  :(

Reply:
This options is mostly for devs and daily development and testing. 

Yes, product wouldn't want to use that but there are a few config that they would probably want to change.

The config could be set the 0 by default but then daily devs will need to enable it constantly or just provision the device..  I can't think of a great solution to please both situations

Reply:
I'll keep it like that, for now, has the common `src/include/platform/CHIPDeviceConfig.h` already use that define for PASE test configs/fallbacks